vmdb/app/assets/javascripts/miq_application.js

```diff
@@ -849,7 +856,7 @@ var miqAjaxTimers = 0;
 
 // Handle an ajax form button press (i.e. Submit) by starting the spinning Q, then waiting for .7 seconds for observers to finish
 function miqAjaxButton(url, serialize_fields) {
-  if (typeof serialize_fields == "unknown") {
+  if (typeof serialize_fields == "undefined") {
```

Comment:
@martinpovolny checked by submitting changes after editing service dialog and by canceling changes in cloud providers editing. Works correctly for me.